docs/docs-components/CombinationNew.js

```diff
@@ -73,6 +80,9 @@ export default function CombinationNew({ children, hideTitle, ...props }: Props)
         shadeColor={cardShadeColor}
         title={hideTitle ? undefined : combinationTitles}
       >
+        <Box position="absolute" top left bottom right>
+          {hasCheckerboard && <Checkerboard />}
```

Comment:
This can addressed in a separate diff, but would it make more sense for Checkerboard to accept children and wrap the content displayed on top of it? That would obviate these `hasCheckerboard` props and allow for more composability

Reply:
Agreed, I took this from our old version, but it could do with a revamp